src/instructlab/model/backends/backends.py

```diff
@@ -122,6 +122,13 @@ def is_model_safetensors(model_path: pathlib.Path) -> bool:
     Returns:
         bool: True if the model is a safetensors model, False otherwise.
     """
+    if not model_path.is_dir():
+        logging.debug(
+            "The path to the model %s is not a directory, and therefore cannot be a model in .safetensors format.",
```

Comment:
Ditto to what Seb said in https://github.com/instructlab/instructlab/pull/1791/files#r1683986353

Reply:
++